packages/analytics/lib/structs.js

```diff
@@ -15,7 +15,7 @@
  */
 import struct from '@react-native-firebase/app/lib/common/struct';
 
-const Item = struct({
+const Item = struct.interface({
```

Comment:
Not sure on just making `struct.interface`, presumably this would make all other properties permissible?

Reply:
No, it accepts arbitrary custom event parameters while rejecting defined parameters with wrong types